lib/src/test/resources/README.MD

```diff
@@ -0,0 +1,88 @@
+Example Resources
```

Comment:
Move this documentation to the docs site. Maybe just create an EXAMPLES section on the index.md page.